Documentation/ApiOverview/ErrorAndExceptionHandling/Configuration/Index.rst

```diff
@@ -10,232 +10,64 @@
 Configuration
 =============
 
-All configuration options related to error and exception handling are
-part of :php:`$GLOBALS['TYPO3_CONF_VARS'][SYS]`:
+You can configure the most important settings for production or debug error
```

Comment:
```suggestion
You can configure the most important settings for live or debug error
```

As the GUI distincts between "live" and "debug" we should stick to that wording:
![image](https://user-images.githubusercontent.com/2566282/200175368-4ef410dc-8e04-4ad0-a864-d052bb13d393.png)

Also: Can we add an image of the GUI here?